### ISSUE TITLE
Indentation from left edge needs more work

### ISSUE THREAD

**Issue by duerst** [rfc-format/draft-iab-rfc-css-bis#10]
Currently, there are (too!!!) many different amounts of indentation from the left edge of the page. That "Index" is less indented than anything else seems to be a clear error. But the bigger problem is that block text is indented more than headings. This may be a direct influence from the text format, but it looks really out of place in the new format where headings stand out clearly because of their larger size and heavier weight. I have just at random checked a wide variety of books in my bookshelves, and none of them indents text blocks more than headings.

**Comment by martinthomson** [comment]
I agree with @duerst here, though Heather seems to take the opposite position in #15.

**Comment by hlflanagan** [comment]
I don't actually have a preference. I was reporting feedback on behalf of someone else in #15
